client/js/components/month/month.js

```diff
@@ -1,12 +1,34 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthCalendarContainer = require('./month-calendarcontainer.js');
+var MonthEventsContainer = require('./month-eventcontainer.js');
+var MonthStore = require('../../stores/month-store.js');
+var MonthHeader = require('./month-header.js');
+var MonthData = require('../../monthData.js');
+
+function monthData() {
+  return {
+    gigs: MonthStore.getMonthData()
+  };
+}
 
 var Month = React.createClass({
-  render: function () {
+  getInitialState: function() {
+    return MonthData;
+  },
+  componentWillMount: function() {
+    MonthStore.addChangeListener(this._onChange);
+  },
+  componentWillUnmount: function() {
+    MonthStore.removeChangeListener(this._onChange);
+  },
+  _onChange: function() {
+    this.setState(monthData());
+  },
+  render: function() {
     return (
       <div className="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
-        <MonthCalendarContainer />
+        <MonthHeader />
+        <MonthEventsContainer monthEvents= { this.state } />
```

Comment:
Seems like the property here should be monthWeeks, per the comment made in month-eventcontainer.js